src/Utils/Html.php

```diff
@@ -126,6 +126,74 @@ public function addAttributes(array $attrs)
 
 
 	/**
+	 * Adds one element's attribute.
+	 * @param  string    HTML attribute name
+	 * @param  mixed     HTML attribute value
+	 * @param  mixed     HTML attribute value's data
+	 * @return self
+	 */
+	public function addAttribute($name, $value, $data = TRUE)
+	{
```

Comment:
What other attributes besides **class** needs value appending? 
Wouldn't it be better to create special methods for **class** attribute - `addClass()`, `removeClass()` and for common attributes keep only basic `setAttribute()`, `getAttribute()` and `removeAttribute()`?